app/routes/questionnaire.py

```diff
@@ -516,32 +542,6 @@ def get_page_title_for_location(schema, current_location, context):
     return safe_content(page_title)
 
 
-def _handle_form_for_block(form, schema, block_handler, current_location):
```

Comment:
I removed this and made it part of the single view method as this is not reused and to make it easier to follow.